index.js

```diff
@@ -38,6 +38,19 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('pre-id', {
```

Comment:
Sorry, @stevemao you are right. The npm publish tag is not the same like the git tag.